Code/Legacy/CryCommon/ILevelSystem.h

```diff
@@ -58,6 +58,9 @@ struct ILevelSystem
 {
     virtual ~ILevelSystem() = default;
 
+#if defined (CARBONATED)
+    virtual void Activate() {}
```

Comment:
Let's rename this to something besides Activate, as that is used/recognized mostly for Components & Entities.

Reply:
Renamed to LoadDefferedLevel()